config/initializers/redis.rb

```diff
@@ -0,0 +1,2 @@
+Redis.sadd_returns_boolean = false
+Redis.raise_deprecations = true if ENV["CI"]
```

Comment:
I think I'd rather do that in test environment rather than checking this variable which we are not currently using anywhere.

Reply:
Done.